crates/host/src/wasmbus/mod.rs

```diff
@@ -177,6 +177,7 @@ impl Queue {
 #[derive(Debug)]
 struct ActorInstance {
     actor: wasmcloud_runtime::Component,
+    #[allow(unused)]
```

Comment:
Should all of these just be removed? Or are we just keeping them around for now? If we're going to remove in a follow up, let's leave a TODO comment so these are easier to find later

Reply:
Will remove in a follow up!